henson_database/__init__.py

```diff
@@ -47,6 +47,7 @@ def from_settings(settings):
     Returns:
         dict: The database-specific settings, formatted to use with
             :func:`connection_url`.
+
```

Comment:
I don't understand why adding these newlines in the docs fixed the issue.  I didn't add newlines in all of the docs (only the ones that showed as broken in the pr builder output).  I'd appreciate it if someone could tell me why that's the case.

Reply:
This is defined in [PEP 257](https://www.python.org/dev/peps/pep-0257/) and checked by `flake8-docstrings`. This [article](http://blog.dolphm.com/pep257-good-python-docstrings-by-example/) provides some good examples.

Reply:
Yes @zackmorris , and we use `tox` to run a PEP linter against any PR. This repository was created before that PEP 257 change, so new PRs are subject to these new additions.

Reply:
Thanks for the answers.  I skimmed those two articles earlier but it still wasn't clear to me why the newline isn't required [here](https://github.com/lvauthrin/Henson-Database/blob/6497d930a4362d06d27717193a3436668937b05d/henson_database/__init__.py#L147).

Also, PEP257 states:
> Insert a blank line after all docstrings (one-line or multi-line) that document a class...

Would that mean a newline should be added [here](https://github.com/lvauthrin/Henson-Database/blob/6497d930a4362d06d27717193a3436668937b05d/henson_database/__init__.py#L73-L83) as well?

Reply:
Hmm, not getting code snippets to work right now.  Sorry for the links above.

Reply:
The PEP257 check only runs on newly submitted pull request and any file those contain. 

Our linter added that check not to long ago, so there are older doc strings that will need an extra line throughout all our repositories.

Reply:
So the check actually ran on a file I didn't change (I had only updated the `docs/index.rst` initially).  The other links I provided in my comment above are from the same file that I had to change to make the pr-builder pass.